theme/static/js/custom.js

```diff
@@ -179,7 +179,9 @@ $(document).ready(function () {
         $(".navbar-inverse .res-dropdown .dropdown-menu").toggleClass("disabled", true);
 
         var formData = new FormData();
-        title = "Untitled Resource";
+        if (!title) {
+            title = "Untitled Resource";
+        }
         formData.append("csrfmiddlewaretoken", csrf_token);
```

Comment:
`title` had to be moved from the constructor, because otherwise the code is not compatible with IE 11 and the entire page doesn't load. Adding the conditional ensures that if the user enters a resource title in the first modal, it is carried through and not overwritten.